sql/mysqld.cc

```diff
@@ -5033,6 +5033,21 @@ a file name for --log-bin-index option", opt_binlog_index_name);
   }
   plugins_are_initialized= TRUE;  /* Don't separate from init function */
 
+  LEX_CSTRING keyring_vault_name= {C_STRING_WITH_LEN("keyring_vault")};
+  LEX_CSTRING keyring_name= {C_STRING_WITH_LEN("keyring_file")};
+  if ((plugin_is_ready(keyring_vault_name, MYSQL_KEYRING_PLUGIN) ||
+       plugin_is_ready(keyring_name, MYSQL_KEYRING_PLUGIN)) &&
+      !pxc_encrypt_cluster_traffic)
+  {
+    WSREP_WARN(
+        "You have keyring plugin enabled. SST encryption is mandatory. "
```

Comment:
Not a strong opinion. But was thinking how about we say 
`SST encryption is mandatory when keyring plugin is enabled. So, please enable pxc_encrypt_cluster_traffic or make sure you have SST encryption configured under [SST] section of your my.cnf - Check https://www.percona.com/doc/percona-xtradb-cluster/5.7/security/encrypt-traffic.html#encrypt-sst for more details.
`

Reply:
@venkatesh-prasad-v will put @patrickbirch in the loop to validate the proper wording.
@patrickbirch can you please provide us with the syntactically correct wording for above warning message?

Reply:
You have enabled the keyring plugin. SST encryption is mandatory.